FinSearchUnified/Tests/Bundle/SearchBundleFindologic/FacetHandler/ImageFacetHandlerTest.php

```diff
@@ -68,48 +68,42 @@ public function imageFilterProvider()
                     ['name' => 'Purple', 'image' => 'https://example.com/purple.gif', 'status' => 404],
                     ['name' => 'Light Purple', 'image' => 'https://example.com/light-purple.gif', 'status' => 404],
                 ],
-                'condition' =>
-                    new ProductAttributeCondition(
-                        'vendor',
-                        Operator::EQ,
-                        ['Red', 'Zima Blue', 'Purple']
-                    ),
-                'facetData' =>
-                    [
-                        ['id' => 'Red', 'active' => true, 'media' => null],
-                        ['id' => 'Green', 'active' => false, 'media' => null],
-                        ['id' => 'Zima Blue', 'active' => true, 'media' => 'https://example.com/zima-blue.gif'],
-                        ['id' => 'Yellow', 'active' => false, 'media' => 'https://example.com/yellow.gif'],
-                        ['id' => 'Purple', 'active' => true, 'media' => null],
-                        ['id' => 'Light Purple', 'active' => false, 'media' => null],
-                    ]
+                'condition' => new ProductAttributeCondition(
+                    'vendor',
+                    Operator::EQ,
+                    ['Red', 'Zima Blue', 'Purple']
+                ),
+                'facetData' => [
+                    ['id' => 'Red', 'active' => true, 'media' => null],
+                    ['id' => 'Green', 'active' => false, 'media' => null],
+                    ['id' => 'Zima Blue', 'active' => true, 'media' => 'https://example.com/zima-blue.gif'],
+                    ['id' => 'Yellow', 'active' => false, 'media' => 'https://example.com/yellow.gif'],
+                    ['id' => 'Purple', 'active' => true, 'media' => 'https://example.com/purple.gif'],
+                    ['id' => 'Light Purple', 'active' => false, 'media' => 'https://example.com/light-purple.gif'],
```

Comment:
Why is there a media url and before it wasn't?

Reply:
Because before this would be `null` if Guzzle wasn't able to receive the image. In the new implementation we're always setting an image.